hypha/apply/funds/templates/funds/modals/update_partner_form.html

```diff
@@ -2,16 +2,6 @@
 
 {% modal_title %}Update partners{% endmodal_title %}
 
-<form
-    class="p-4 form"
-    id="partner_form"
-    hx-target="this"
-    hx-swap="outerHTML"
-    method="POST"
-    hx-post="{% url 'funds:submissions:partners_update' pk=object.pk %}"
->
-    {% csrf_token %}
-
-    {% url 'funds:submissions:partners_update' pk=object.pk as progress_url %}
-    {% include 'funds/includes/dialog_form_base.html' with form=form value=value url=progress_url %}
-</form>
```

Comment:
@theskumar After removing this the forms still seems to work just as before. I'm I missing something?

Wrapping a form in a form feels wrong as well. The `url` attribute I can not find being used in `dialog_form_base.html`. Some other modals on the project side has this extra code as well.

Reply:
@theskumar Ping